drivers/amazonec2/amazonec2.go

```diff
@@ -610,6 +593,17 @@ func (d *Driver) isSwarmMaster() bool {
 	return d.SwarmMaster
 }
 
+func (d *Driver) securityGroupAvailableFunc(id string) func() bool {
```

Comment:
Hm, I don't really care either way but I called it that way to be more clear to the reader of the code that it actually returns a function as opposed to the others which return a boolean.  Happy to take it out if you think it's confusing or redundant.